community/OM-Operator-Management/cnv-samples/virtual-machines/fedora-rootdisk-and-datadisk.yaml

```diff
@@ -0,0 +1,229 @@
+                apiVersion: kubevirt.io/v1
+                kind: VirtualMachine
+                metadata:
+                  name: vm02
+                  namespace: default
+                spec:
+                  dataVolumeTemplates:
+                    - metadata:
+                        creationTimestamp: null
+                        name: fedora-vm02
+                      spec:
+                        sourceRef:
+                          kind: DataSource
+                          name: fedora
+                          namespace: openshift-virtualization-os-images
+                        storage:
+                          resources:
+                            requests:
+                              storage: '32212254720'
+                          storageClassName: managed-csi
+                    - metadata:
+                        creationTimestamp: null
+                        name: dv-fedora-vm02
+                      spec:
+                        source:
+                          blank: {}
+                        storage:
+                          resources:
+                            requests:
+                              storage: 30Gi
+                          storageClassName: managed-csi
+                  instancetype:
+                    kind: virtualmachineclusterinstancetype
+                    name: u1.small
+                  preference:
+                    kind: virtualmachineclusterpreference
+                    name: fedora
+                  runStrategy: Halted
+                  template:
+                    metadata:
+                      creationTimestamp: null
+                      labels:
+                        network.kubevirt.io/headlessService: headless
+                    spec:
+                      architecture: amd64
+                      domain:
+                        devices:
+                          autoattachPodInterface: false
+                          disks:
+                            - disk:
+                                bus: virtio
+                              name: disk-vm02
+                          interfaces:
+                            - masquerade: {}
+                              name: default
+                        machine:
+                          type: pc-q35-rhel9.4.0
+                        resources: {}
+                      networks:
+                        - name: default
+                          pod: {}
+                      subdomain: headless
+                      volumes:
+                        - dataVolume:
+                            name: fedora-vm02
+                          name: rootdisk
+                        - cloudInitNoCloud:
+                            userData: |
+                              #cloud-config
+                              chpasswd:
+                                expire: false
+                              password: change-me-5738
```

Comment:
This password will be saved in plaintext in etcd, which might be a security concern. Since this is an example, that might not be a problem, but it could be worth mentioning. To keep it secure, it could use templates - in particular, a hub template `fromSecret` should keep the value encrypted throughout all of the Policies and ConfigurationPolicy.